structuredheadings/plugin.js

```diff
@@ -464,6 +466,113 @@
             setStyle(editor, node, editor.config.autonumberCurrentStyle);
           }
         }
+      },
+
+      /*
+       * convertList
+       */
+      convertList: {
+        exec: function (editor) {
+
+          var selectArea = function (startElement, endElement, encompass) {
+            encompass = encompass ? encompass : false;
+            var range = editor.createRange();
+            if (encompass) {
+              range.setStartBefore(startElement, 0);
+              range.setEndAfter(endElement, 0);
+            } else {
+              range.setStart(startElement, 0);
+              range.setEnd(endElement, 0);
```

Comment:
I'm not sure, but given the 0 index on the startElement's offset , i would have expected setEnd to be the last-1 offset of endElement.  Maybe this needs more explanation